.gitignore

```diff
@@ -53,3 +53,6 @@ test/gutenberg-test-themes/twentytwentyone
 test/gutenberg-test-themes/twentytwentythree
 test/gutenberg-test-themes/twentytwentyfour
 packages/react-native-editor/src/setup-local.js
+
+# JetBrains IDE Generated Files (WebStorm)
+.idea
```

Comment:
You can use the global `excludesfile` git configuration to ignore your platform/IDE-specific files - https://gist.github.com/subfuzion/db7f57fff2fb6998a16c. It might be even more than `.idea`.

Reply:
It's only ever `.idea`, JetBrains stores all their IDE related stuff in that folder so it's easy to exclude. With that said, I've reverted the `.gitignore`